diff --git TPL/libsvm-3.1-custom/svm.cpp TPL/libsvm-3.1-custom/svm.cpp
index b5ae77c..1f01fcc 100644
--- TPL/libsvm-3.1-custom/svm.cpp
+++ TPL/libsvm-3.1-custom/svm.cpp
@@ -37,6 +37,13 @@ static inline double powi(double base, int times)
 #define TAU 1e-12
 #define Malloc(type,n) (type *)malloc((n)*sizeof(type))
 
+// Calculates log2 of number.
+static inline float log2( float n )
+{
+  // log(n)/log(2) is log2.
+  return log( n ) / log( 2.0f );
+}
+
 static void print_string_stdout(const char *s)
 {
   fputs(s,stdout);
