--- xml/xpath/ParsedAbbreviatedAbsoluteLocationPath.py	2003-03-11 14:01:34.000000000 +0000
+++ xml/xpath/ParsedAbbreviatedAbsoluteLocationPath.py.new	2015-05-13 16:01:34.855434243 +0000
@@ -24,8 +24,8 @@
         self._rel = rel
         nt = ParsedNodeTest.ParsedNodeTest('node', '')
         ppl = ParsedPredicateList.ParsedPredicateList([])
-        as = ParsedAxisSpecifier.ParsedAxisSpecifier('descendant-or-self')
-        self._step = ParsedStep.ParsedStep(as, nt, ppl)
+        axs = ParsedAxisSpecifier.ParsedAxisSpecifier('descendant-or-self')
+        self._step = ParsedStep.ParsedStep(axs, nt, ppl)
         return
 
     def evaluate(self, context):
