diff --git setup.py setup.py
index 2d8cafa..f95c871 100644
--- setup.py
+++ setup.py
@@ -575,7 +575,7 @@ class pil_build_ext(build_ext):
             exts.append(Extension(
                 "PIL._webp", ["_webp.c"], libraries=libs, define_macros=defs))
 
-        if sys.platform == "darwin":
+        if sys.platform == "":
             # locate Tcl/Tk frameworks
             frameworks = []
             framework_roots = [
