diff --git setup.py setup.py
index cc76e65..45fc1c3 100644
--- setup.py
+++ setup.py
@@ -55,7 +55,7 @@ setup(
         "pattern.text.it"         : ["*.txt", "*.xml"],
         "pattern.text.nl"         : ["*.txt", "*.xml"],
         "pattern.vector"          : ["*.txt"],
-        "pattern.vector.svm"      : ["*.txt", "libsvm-3.11/*", "libsvm-3.17/*", "liblinear-1.93/*"],
+        "pattern.vector.svm"      : ["*.txt"],
         "pattern.graph"           : ["*.js", "*.csv"],
     },
     py_modules = [
@@ -87,4 +87,4 @@ setup(
         "Topic :: Text Processing :: Linguistic",
         "Topic :: Text Processing :: Markup :: HTML"
     ]
-)
\ No newline at end of file
+)
