--- share/make/vars.mk.orig	2014-01-04 04:33:29.000000000 -0500
+++ share/make/vars.mk	2014-01-04 04:33:41.000000000 -0500
@@ -9,6 +9,6 @@
 ## Those with standard R directories (not datasets, methods)
 R_PKGS_BASE2 = base tools utils grDevices graphics stats grid splines stats4 tcltk compiler parallel
 
-R_PKGS_RECOMMENDED =  MASS lattice Matrix nlme survival boot cluster codetools foreign KernSmooth rpart class nnet spatial mgcv
+R_PKGS_RECOMMENDED =
 # there are dependencies in src/library/Recommended/Makefile*
 # which this order respects
