diff -urN ld64.orig/src/ld/parsers/macho_relocatable_file.cpp ld64/src/ld/parsers/macho_relocatable_file.cpp
--- ld64.orig/src/ld/parsers/macho_relocatable_file.cpp	2013-10-07 17:09:08.975777780 +0100
+++ ld64/src/ld/parsers/macho_relocatable_file.cpp	2013-10-07 17:09:09.149113735 +0100
@@ -51,6 +51,11 @@
 #include "macho_relocatable_file.h"
 
 
+#ifdef HAVE_BSD_QSORT_R
+ #define QSORT_R(_a,_b,_c,_vp,_fn) ::qsort_r((_a), (_b), (_c), (_vp), (_fn))
+#else
+ #define QSORT_R(_a,_b,_c,_vp,_fn) ::qsort_r((_a), (_b), (_c), (int(*)(const void*, const void*, void*))(_fn), (_vp))
+#endif
 
 extern void throwf(const char* format, ...) __attribute__ ((noreturn,format(printf, 1, 2)));
 extern void warning(const char* format, ...) __attribute__((format(printf, 1, 2)));
@@ -1058,8 +1063,13 @@
 	void											makeSortedSymbolsArray(uint32_t symArray[], const uint32_t sectionArray[]);
 	void											makeSortedSectionsArray(uint32_t array[]);
 	static int										pointerSorter(const void* l, const void* r);
+#ifdef HAVE_BSD_QSORT_R
 	static int										symbolIndexSorter(void* extra, const void* l, const void* r);
 	static int										sectionIndexSorter(void* extra, const void* l, const void* r);
+#else
+	static int										symbolIndexSorter(const void* l, const void* r, void* extra);
+	static int										sectionIndexSorter(const void* l, const void* r, void* extra);
+#endif
 
 	void											parseDebugInfo();
 	void											parseStabs();
@@ -1841,8 +1851,13 @@
 	}
 }
 
+#ifdef HAVE_BSD_QSORT_R
 template <typename A>
 int Parser<A>::sectionIndexSorter(void* extra, const void* l, const void* r)
+#else
+template <typename A>
+int Parser<A>::sectionIndexSorter(const void* l, const void* r, void* extra)
+#endif
 {
 	Parser<A>* parser = (Parser<A>*)extra;
 	const uint32_t* left = (uint32_t*)l;
@@ -1885,7 +1900,7 @@
 	// sort by symbol table address
 	for (uint32_t i=0; i < _machOSectionsCount; ++i)
 		array[i] = i;
-	::qsort_r(array, _machOSectionsCount, sizeof(uint32_t), this, &sectionIndexSorter);
+	QSORT_R(array, _machOSectionsCount, sizeof(uint32_t), this, sectionIndexSorter);
 
 	if ( log ) {
 		fprintf(stderr, "sorted sections:\n");
@@ -1896,8 +1911,13 @@
 
 
 
+#ifdef HAVE_BSD_QSORT_R
 template <typename A>
 int Parser<A>::symbolIndexSorter(void* extra, const void* l, const void* r)
+#else
+template <typename A>
+int Parser<A>::symbolIndexSorter(const void* l, const void* r, void* extra)
+#endif
 {
 	ParserAndSectionsArray* extraInfo = (ParserAndSectionsArray*)extra;
 	Parser<A>* parser = extraInfo->parser;
@@ -1969,7 +1989,7 @@
 	
 	// sort by symbol table address
 	ParserAndSectionsArray extra = { this, sectionArray };
-	::qsort_r(array, _symbolsInSections, sizeof(uint32_t), &extra, &symbolIndexSorter);
+	QSORT_R(array, _symbolsInSections, sizeof(uint32_t), &extra, symbolIndexSorter);
 	
 	// look for two symbols at same address
 	_overlappingSymbols = false;
