--- ld64.orig/src/ld/parsers/macho_relocatable_file.cpp.orig	2017-08-06 15:00:16.416272570 +0100
+++ ld64/src/ld/parsers/macho_relocatable_file.cpp	2017-08-06 15:01:30.358566505 +0100
@@ -60,12 +60,7 @@
 extern void throwf(const char* format, ...) __attribute__ ((noreturn,format(printf, 1, 2)));
 extern void warning(const char* format, ...) __attribute__((format(printf, 1, 2)));
 
-// Work around for an old compiler bug.
-#if (__GNUC__>=4) && (__GNUC_MINOR__>=5)
- #define _TYPENAME typename
-#else
- #define _TYPENAME
-#endif
+#define _TYPENAME typename
 
 namespace mach_o {
 namespace relocatable {
