--- IPython/frontend/html/notebook/handlers.py
+++ Ipython/frontend/html/notebook/handlers.py
@@ -228,7 +228,7 @@
             if passwd_check(self.application.password, pwd):
                 self.set_secure_cookie('username', str(uuid.uuid4()))
             else:
-                self._render(message={'error': 'Invalid password'})
+                self._render(message={'error': 'Invalid password. This is your Wakari password.'})
                 return
 
         self.redirect(self.get_argument('next', default=self.application.ipython_app.base_project_url))

--- IPython/frontend/html/notebook/static/js/kernel.js
+++ Ipython/frontend/html/notebook/static/js/kernel.js
@@ -81,6 +81,7 @@
         this.running = true;
         this.kernel_id = json.kernel_id;
         this.ws_url = json.ws_url;
+        this.ws_url = this.ws_url.replace("ws://", "wss://")   // Wakari Patch
         this.kernel_url = this.base_url + "/" + this.kernel_id;
         this.start_channels();
         this.shell_channel.onmessage = $.proxy(this._handle_shell_reply,this);
