--- pymssql-2.1.3/setup.py.orig	2016-09-03 13:39:04.565627400 +0100
+++ pymssql-2.1.3/setup.py	2016-09-04 12:20:09.687056500 +0100
@@ -290,7 +290,7 @@
                     freetds_dir = 'vs2008'
                 if LINK_FREETDS_STATICALLY:
                     libraries = [
-                        'db-lib', 'tds', 'replacements',
+                        'libsybdb', 'tds', 'replacements',
                         'iconv',
                         'ws2_32', 'wsock32', 'kernel32', 'shell32',
                     ]
