From d196e78c60f4b471a40bff85aec40b1ca2f03e51 Mon Sep 17 00:00:00 2001
From: Mark Wiebe <mwwiebe@gmail.com>
Date: Sat, 1 Sep 2012 22:50:53 -0700
Subject: [PATCH] 'b'


diff --git nmake.opt nmake.opt
index c136c36..df324a3 100644
--- nmake.opt
+++ nmake.opt
@@ -63,10 +63,10 @@ LOGLUV_SUPPORT	= 1
 # Uncomment and edit following lines to enable ZIP support
 # (required for Deflate compression and Pixar log-format)
 #
-#ZIP_SUPPORT	= 1
+ZIP_SUPPORT	= 1
 #ZLIBDIR 	= d:/projects/zlib-1.2.1
-#ZLIB_INCLUDE	= -I$(ZLIBDIR)
-#ZLIB_LIB 	= $(ZLIBDIR)/zlib.lib
+ZLIB_INCLUDE	=
+ZLIB_LIB 	= zlibstatic.lib
 
 #
 # Uncomment and edit following lines to enable ISO JBIG support
-- 
1.7.8.msysgit.0

