diff --git livereload/server.py livereload/server.py
index e848bf1..48187d0 100644
--- livereload/server.py
+++ livereload/server.py
@@ -5,6 +5,7 @@
 Core Server of LiveReload.
 """

+import sys
 import os
 import logging
 import time
@@ -22,6 +23,10 @@ except ImportError:
     from tornado.options import enable_pretty_logging
 from livereload.task import Task

+if sys.version_info[0] == 3:
+    text_type = str
+else:
+    text_type = unicode

 PORT = 35729
 ROOT = '.'
@@ -164,6 +169,8 @@ class IndexHandler(RequestHandler):
                 self.write(data)

             hasher = hashlib.sha1()
+            if isinstance(data, text_type):
+                data = data.encode("utf-8")
             hasher.update(data)
             self.set_header('Etag', '"%s"' % hasher.hexdigest())
             return
