diff --git config/Makefile.linux config/Makefile.linux
index b460b4b..ae45ead 100644
--- config/Makefile.linux
+++ config/Makefile.linux
@@ -11,7 +11,7 @@ ifeq (ppc64,${M_ARCH})
 endif
 ifeq (${ARCH64},true)
   LDFLAGS.EXTRA = -L/usr/X11R6/lib64 -L/usr/lib64
-  LIBDIR = $(GLEW_DEST)/lib64
+  LIBDIR = $(GLEW_DEST)/lib
 else
   LDFLAGS.EXTRA = -L/usr/X11R6/lib -L/usr/lib
   LIBDIR = $(GLEW_DEST)/lib
