--- src/font.c.old	2015-11-11 15:31:48 -0500
+++ src/font.c	2015-11-11 15:34:19 -0500
@@ -89,8 +89,9 @@
 };
 */
 
+#define DEFERRED(X) 0
 PyTypeObject PycairoFontFace_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(DEFERRED(&PyType_Type), 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.FontFace",                   /* tp_name */
@@ -141,6 +142,7 @@
 static PyObject *
 toy_font_face_new (PyTypeObject *type, PyObject *args, PyObject *kwds) {
   const char *utf8;
+  PyObject *o;
   cairo_font_slant_t slant   = CAIRO_FONT_SLANT_NORMAL;
   cairo_font_weight_t weight = CAIRO_FONT_WEIGHT_NORMAL;
 
@@ -148,7 +150,7 @@
 			 "utf-8", &utf8, &slant, &weight))
     return NULL;
 
-  PyObject *o = PycairoFontFace_FromFontFace (
+  o = PycairoFontFace_FromFontFace (
 		     cairo_toy_font_face_create (utf8, slant, weight));
   PyMem_Free((void *)utf8);
   return o;
@@ -178,7 +180,7 @@
 
 
 PyTypeObject PycairoToyFontFace_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(DEFERRED(&PyType_Type), 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.ToyFontFace",                /* tp_name */
@@ -340,7 +342,7 @@
 
 
 PyTypeObject PycairoScaledFont_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(DEFERRED(&PyType_Type), 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.ScaledFont",                 /* tp_name */
@@ -527,7 +529,7 @@
 
 
 PyTypeObject PycairoFontOptions_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(DEFERRED(&PyType_Type), 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.FontOptions",                /* tp_name */
