--- src/surface.c.old	2015-11-11 15:54:26 -0500
+++ src/surface.c	2015-11-11 15:57:39 -0500
@@ -345,9 +345,9 @@
   {NULL, NULL, 0, NULL},
 };
 
-
+#define DEFERRED(X) 0
 PyTypeObject PycairoSurface_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(DEFERRED(&PyType_Type), 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.Surface",                    /* tp_name */
@@ -463,6 +463,7 @@
 static cairo_status_t
 _read_func (void *closure, unsigned char *data, unsigned int length) {
   char *buffer;
+  int ret;
   Py_ssize_t str_length;
   cairo_status_t status = CAIRO_STATUS_READ_ERROR;
   PyGILState_STATE gstate = PyGILState_Ensure();
@@ -474,7 +475,7 @@
      */
     goto end;
   }
-  int ret = PyBytes_AsStringAndSize(pyBytes, &buffer, &str_length);
+  ret = PyBytes_AsStringAndSize(pyBytes, &buffer, &str_length);
   if (ret == -1 || str_length < length) {
     goto end;
   }
@@ -646,7 +647,7 @@
 
 
 PyTypeObject PycairoImageSurface_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(DEFERRED(&PyType_Type), 0)
   //  PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.ImageSurface",               /* tp_name */
@@ -771,7 +772,7 @@
 };
 
 PyTypeObject PycairoPDFSurface_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(DEFERRED(&PyType_Type), 0)
   //  PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.PDFSurface",                 /* tp_name */
@@ -912,9 +913,10 @@
 static PyObject *
 ps_surface_ps_level_to_string (PyObject *self, PyObject *args) {
   int level;
+  const char *s;
   if (!PyArg_ParseTuple(args, "i:ps_level_to_string", &level))
     return NULL;
-  const char *s = cairo_ps_level_to_string (level);
+  s = cairo_ps_level_to_string (level);
   if (s == NULL){
     PyErr_SetString(CairoError, "ps_level_to_string: "
 		    "invalid level argument");
@@ -974,7 +976,7 @@
 };
 
 PyTypeObject PycairoPSSurface_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(DEFERRED(&PyType_Type), 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.PSSurface",                  /* tp_name */
@@ -1089,7 +1091,7 @@
 };
 
 PyTypeObject PycairoSVGSurface_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(DEFERRED(&PyType_Type), 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.SVGSurface",                 /* tp_name */
@@ -1155,7 +1157,7 @@
 };
 
 PyTypeObject PycairoWin32Surface_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(DEFERRED(&PyType_Type), 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.Win32Surface",               /* tp_name */
@@ -1218,7 +1220,7 @@
 };
 
 PyTypeObject PycairoWin32PrintingSurface_Type = {
-  PyVarObject_HEAD_INIT(&PyType_Type, 0)
+  PyVarObject_HEAD_INIT(DEFERRED(&PyType_Type), 0)
   //PyObject_HEAD_INIT(NULL)
   //0,                                  /* ob_size */
   "cairo.Win32PrintingSurface",       /* tp_name */
