diff --git a/test/inp/B03.py b/test/inp/B03.py
index f80cb46..404782d 100644
--- test/inp/B03.py
+++ test/inp/B03.py
@@ -27,9 +27,9 @@ def load():
          cmd.rewind()
          cmd.orient('pdb')
          cmd.refresh()
-         cmd.as("ribbon")
+#         cmd.as("ribbon")
          cmd.refresh()
-         cmd.as("sticks")
+#         cmd.as("sticks")
          cmd.refresh()
          sys.__stderr__.write(".")
          sys.__stderr__.flush()
diff --git a/test/inp/B05.py b/test/inp/B05.py
index c3af086..d1cdccb 100644
--- test/inp/B05.py
+++ test/inp/B05.py
@@ -26,7 +26,7 @@ def load():
             name = string.split(name,'.')[0]
             cmd.disable()
             cmd.load(file,name)
-            cmd.as("cartoon",name)
+#            cmd.as("cartoon",name)
             cmd.refresh()
             cmd.dss(name)
             cmd.refresh()
diff --git a/test/inp/B11.py b/test/inp/B11.py
index 1008325..ddd33ea 100644
--- test/inp/B11.py
+++ test/inp/B11.py
@@ -43,7 +43,7 @@ def load():
                z = z + 1
    cmd.zoom()
    cmd.set("sphere_mode",1)
-   cmd.as("spheres")
+#   cmd.as("spheres")
    cmd.rebuild()
    cmd.set("hash_max",250)
    
