diff --git setupext.py setupext.py
index 9219c88..1eec994 100644
--- setupext.py
+++ setupext.py
@@ -1329,9 +1329,10 @@ class BackendTkAgg(OptionalBackendPackage):
     def add_flags(self, ext):
         if sys.platform == 'win32':
             major, minor1, minor2, s, tmp = sys.version_info
-            ext.include_dirs.extend(['win32_static/include/tcl85'])
+            tcl_dir = r'C:\Tcl%d' % (8 * tuple.__itemsize__)
+            ext.include_dirs.append(os.path.join(tcl_dir, 'include'))
+            ext.library_dirs.append(os.path.join(tcl_dir, 'lib'))
             ext.libraries.extend(['tk85', 'tcl85'])
-            ext.library_dirs.extend([os.path.join(sys.prefix, 'dlls')])
 
         elif sys.platform == 'darwin':
             # this config section lifted directly from Imaging - thanks to
