From 82fff54a7173ccd858537269b644509d4f60c96d Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Tue, 9 Aug 2016 11:49:59 +0100
Subject: [PATCH 05/42] Debugging

---
 src/cpp/desktop/DesktopSessionLauncher.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/cpp/desktop/DesktopSessionLauncher.cpp b/src/cpp/desktop/DesktopSessionLauncher.cpp
index 40bb085..347a5b6 100644
--- a/src/cpp/desktop/DesktopSessionLauncher.cpp
+++ b/src/cpp/desktop/DesktopSessionLauncher.cpp
@@ -99,6 +99,7 @@ Error SessionLauncher::launchFirstSession(const QString& filename,
    logEnvVar("PATH");
    logEnvVar("HOME");
    logEnvVar("R_USER");
+    RUN_DIAGNOSTICS_LOG("\nSessionLauncher::launchFirstSession .. filename: " + filename.toStdString() + "\n");
 
    // launch the process
    Error error = launchSession(argList, &pRSessionProcess_);
@@ -290,6 +291,9 @@ Error SessionLauncher::launchSession(const QStringList& argList,
    if (error)
       LOG_ERROR(error);
 
+   RUN_DIAGNOSTICS_LOG("\nSessionLauncher::launch: " + sessionPath_.absolutePath() + "\n");
+
+
    return  parent_process_monitor::wrapFork(
          boost::bind(launchProcess,
                      sessionPath_.absolutePath(),
-- 
2.10.2

