diff --git boost/archive/iterators/transform_width.hpp boost/archive/iterators/transform_width.hpp
index 3562144..f95c675 100644
--- boost/archive/iterators/transform_width.hpp
+++ boost/archive/iterators/transform_width.hpp
@@ -30,6 +30,8 @@
 #include <boost/iterator/iterator_adaptor.hpp>
 #include <boost/iterator/iterator_traits.hpp>
 
+#include <algorithm>
+
 namespace boost { 
 namespace archive {
 namespace iterators {
-- 
1.8.3.msysgit.0

