diff --git pkg/R.frame.pkg/R.framework/Versions/3.2/Resources/bin/R pkg/R.frame.pkg/R.framework/Versions/3.2/Resources/bin/R
index b289687..e5a9ffc 100755
--- pkg/R.frame.pkg/R.framework/Versions/3.2/Resources/bin/R
+++ pkg/R.frame.pkg/R.framework/Versions/3.2/Resources/bin/R
@@ -1,42 +1,11 @@
 #!/bin/sh
 # Shell wrapper for R executable.
 
-R_HOME_DIR=/Library/Frameworks/R.framework/Resources
-if test "${R_HOME_DIR}" = "/Library/Frameworks/lib/R"; then
-   case "darwin14.5.0" in
-   linux*)
-     run_arch=`uname -m`
-     case "$run_arch" in
-        x86_64|mips64|ppc64|powerpc64|sparc64|s390x)
-          libnn=lib64
-          libnn_fallback=lib
-        ;;
-        *)
-          libnn=lib
-          libnn_fallback=lib64
-        ;;
-     esac
-     if [ -x "/Library/Frameworks/${libnn}/R/bin/exec/R" ]; then
-        R_HOME_DIR="/Library/Frameworks/${libnn}/R"
-     elif [ -x "/Library/Frameworks/${libnn_fallback}/R/bin/exec/R" ]; then
-        R_HOME_DIR="/Library/Frameworks/${libnn_fallback}/R"
-     ## else -- leave alone (might be a sub-arch)
-     fi
-     ;;
-  esac
-fi
-
-if test -n "${R_HOME}" && \
-   test "${R_HOME}" != "${R_HOME_DIR}"; then
-  echo "WARNING: ignoring environment value of R_HOME"
-fi
-R_HOME="${R_HOME_DIR}"
-export R_HOME
-R_SHARE_DIR=/Library/Frameworks/R.framework/Resources/share
+R_SHARE_DIR=${R_HOME}/share
 export R_SHARE_DIR
-R_INCLUDE_DIR=/Library/Frameworks/R.framework/Resources/include
+R_INCLUDE_DIR=${R_HOME}/include
 export R_INCLUDE_DIR
-R_DOC_DIR=/Library/Frameworks/R.framework/Resources/doc
+R_DOC_DIR=${R_HOME}/doc
 export R_DOC_DIR
 
 # Since this script can be called recursively, we allow R_ARCH to
