--- python-sybase-0.40.orig/setup.py	2016-09-04 23:01:43.372006200 +0100
+++ python-sybase-0.40/setup.py	2016-09-04 23:17:30.327161100 +0100
@@ -66,6 +66,7 @@
 if sys.maxint > 2147483647:
     have64bit = True
 
+haveFreeTDS = 'HAVE_FREETDS' in ''.join(sys.argv)
 if os.name == 'posix':                  # unix
     # Most people will define the location of their Sybase
     # installation in their environment.
@@ -108,7 +109,10 @@
                 'Please define the Sybase installation directory in'
                 'the SYBASE environment variable.\n')
             sys.exit(1)
-    syb_libs = ['libblk', 'libct', 'libcs']
+    if haveFreeTDS:
+        syb_libs = ['libct', 'tds', 'replacements', 'ws2_32', 'wsock32', 'libeay32', 'ssleay32', 'shell32']
+    else:
+        syb_libs = ['libblk', 'libct', 'libcs']
     # This seems a bit sloppy to me, but is no worse than what's above.
     if sybase.find('15') > 0:
         syb_libs = ['libsybblk', 'libsybct', 'libsybcs']
