diff --git a/siputils.py b/siputils.py
index f0c7323..909ded1 100644
--- siputils.py
+++ siputils.py
@@ -1561,8 +1561,8 @@ class ModuleMakefile(Makefile):
             # Note that I can't remember why we need a framework build.
             dl = get_python_inc().split(os.sep)

-            if "Python.framework" not in dl:
-                error("SIP requires Python to be built as a framework")
+            # if "Python.framework" not in dl:
+            #     error("SIP requires Python to be built as a framework")

             self.LFLAGS.append("-undefined dynamic_lookup")
