diff --git a/MANIFEST.in b/MANIFEST.in
--- MANIFEST.in
+++ MANIFEST.in
@@ -1,5 +1,6 @@
 recursive-include docs *
 recursive-include cloud_sptheme/themes *
+recursive-include cloud_sptheme/ext *
 include LICENSE CHANGES README
 prune docs/_build
 exclude .hgignore *.komodoproject
diff --git a/setup.py b/setup.py
--- setup.py
+++ setup.py
@@ -24,7 +24,7 @@
 setup(
     #package info
     packages = find_packages(),
-    package_data = { "cloud_sptheme": ["themes/*/*.*", "themes/*/static/*.*"] },
+    package_data = { "cloud_sptheme": ["themes/*/*.*", "themes/*/static/*.*", "ext/*"] },
     zip_safe=False,

     install_requires=[ "sphinx>=1.1"],
