diff --git pyOpt/__init__.py pyOpt/__init__.py
index 7177ca8..a4c02f9 100644
--- pyOpt/__init__.py
+++ pyOpt/__init__.py
@@ -14,6 +14,8 @@ from pyOpt_optimizer import Optimizer
 __all__ = ['History','Parameter','Variable','Gradient','Constraint','Objective','Optimization','Optimizer']
 
 dir = os.path.dirname(os.path.realpath(__file__))
+if sys.platform == 'win32':
+    os.environ["PATH"] = os.environ["PATH"] + os.pathsep + dir
 for f in os.listdir(dir):
     if f.startswith('py') and os.path.isdir(os.path.join(dir,f)):
         try:
@@ -24,3 +26,5 @@ for f in os.listdir(dir):
         #end
     #end
 #end
+if sys.platform == 'win32':
+    os.environ["PATH"] = os.environ["PATH"].replace(os.pathsep + dir, '')
\ No newline at end of file
