diff --git Source/LibJXR/image/sys/strcodec.h Source/LibJXR/image/sys/strcodec.h
index 4715b6b..8ee27f2 100755
--- Source/LibJXR/image/sys/strcodec.h
+++ Source/LibJXR/image/sys/strcodec.h
@@ -74,7 +74,9 @@
 #include "arm.h"
 #endif
 
-#ifdef __ANSI__
+//#ifdef __ANSI__                          // Changed based on this suggestion.
+                                           // ( http://sourceforge.net/p/freeimage/discussion/36111/thread/a9cc87b6/#1c28 )
+#if defined(__ANSI__) || defined(__MINGW32__)
 #define PLATFORM_ANSI
 #include "ansi.h"
 #endif
