--- configure.py	2014-09-12 17:58:40.000000000 -0400
+++ configure.py	2014-10-09 11:49:46.000000000 -0400
@@ -707,10 +707,9 @@
 
             config_args = ducfg.get('CONFIG_ARGS', '')
 
-            if sys.platform == 'darwin':
+            dynamic_pylib = '--enable-shared' in config_args
+            if not dynamic_pylib:
                 dynamic_pylib = '--enable-framework' in config_args
-            else:
-                dynamic_pylib = '--enable-shared' in config_args
 
             if dynamic_pylib:
                 pyshlib = ducfg.get('LDLIBRARY', '')
@@ -729,6 +728,7 @@
                     pylib_dir = ''
             else:
                 pyshlib = ''
+                pylibdir = ''
 
         self.py_pylib_dir = pylib_dir
         self.py_pylib_lib = pylib_lib
